src/poetry/console/commands/build.py

```diff
@@ -23,11 +30,13 @@ class BuildCommand(EnvCommand):
     def handle(self) -> None:
         from poetry.core.masonry.builder import Builder
 
-        fmt = self.option("format") or "all"
-        package = self.poetry.package
-        self.line(
-            f"Building <c1>{package.pretty_name}</c1> (<c2>{package.version}</c2>)"
-        )
+        env: Env
```

Comment:
It was complaining when I tried it last, I should do that again to see if anything has changed since the last time with all the other changes going on.